### PR TITLE
Ryan

### PR DESCRIPTION
**Pull request by RyanPWalker**
Added Table for dice card.  Added leave room onclick.  Added how to play link.

### CODE REVIEW

File: modules/Game/index.jsx [hunk 1]
```diff
@@ -32,14 +33,44 @@ const Game = ({ socket }) => {
   //   setMetaState((prev) => ({ ...prev, ...userState }));
   // });
 
+  const leaveRoom = () => {
+    router.push('/lobby');
+  }
+
+  const isChameleon = false;
```
Comment (craig1123) [inline modules/Game/index.jsx:40]: I actually have this data all setup from the sockets, just need to get it. I guess I can do that in a later PR

File: modules/Login/index.jsx [hunk 2]
```diff
@@ -49,6 +49,7 @@ const Login = ({ socket }) => {
         <Button type="submit" variant="primary" size="large">
           Enter Lobby
         </Button>
+        <a href="https://www.youtube.com/watch?v=3IEEUcG0nSo" target="_blank" style={{ float: 'right' }}>How to play?</a>
```
Comment (craig1123) [inline modules/Login/index.jsx:52]: whenever there is a target="blank" on an <a href>, make sure you add `rel="noopener noreferrer"` as an attribute for security reasons

Comment (craig1123) [reply modules/Login/index.jsx:52]: should we add this as an iframe?

Comment (RyanPWalker) [reply modules/Login/index.jsx:52]: nah I think new tab since it's external

File: modules/Game/index.jsx (hunk 1 above)
Comment (RyanPWalker) [reply modules/Game/index.jsx:40]: I can look into that tonight

Comment (craig1123) [reply modules/Game/index.jsx:40]: I'll finish it this afternoon for you. You can work on the styles of the grid

**Review by craig1123 — APPROVED** [review]

**Pull request merged**